### ISSUE TITLE
suggest：Add "Save Configuration" feature to landscape page

### ISSUE THREAD

**Issue by kirklin** [jhipster/jhipster-lite#4872]
I suggest adding a "Save Configuration" feature to the landscape page. This feature would allow users to save their adjusted configurations for future use.

One way to implement this feature would be to use local storage (such as cookies or HTML5's localStorage). This would allow users to save their configurations without the need for a backend API.

In addition to the "Save Configuration" feature, it would also be helpful to provide a "Load Configuration" feature, so that users can easily load their saved configurations at a later time. This would allow users to quickly restore their previously used configurations without the need for manual adjustment.

Overall, adding a "Save Configuration" and "Load Configuration" feature would make the landscape page more convenient and user-friendly, and provide a better user experience.

**Comment by DamnClin** [comment]
Does this take precedence over the configuration loaded from the project?

**Comment by kirklin** [comment]
Yes, this takes precedence over the configuration loaded from the project.

**Comment by simran-bhadani3** [comment]
Hello,
Is this issue accepting PRs?

**Comment by simran-bhadani3** [comment]
Hello @pascalgrimaud,
Is this issue still accepting PRs?
